src/Angular.js

```diff
@@ -1915,6 +1915,7 @@ function bindJQuery() {
   // AngularJS 1.2+ requires jQuery 1.7+ for on()/off() support.
   // AngularJS 1.3+ technically requires at least jQuery 2.1+ but it may work with older
   // versions. It will not work for sure with jQuery <1.7, though.
+  // AngularJS 1.8+ technically requires jQuery 3.5.1+ to mitigate a security issue.
```

Comment:
I'd keep support at 2.1+, maybe with a note that you need 3.5.1+ to mitigate a security issue. I'd like us to keep testing on 2.1+, otherwise the update to 1.8 may be much harder.

Reply:
OK